set.mm

```diff
@@ -645588,6 +645803,128 @@ with respect to a linear (1-dimensional) relation.  (Contributed by BJ
       VDCVBVAJSDVBTZFVDTZULUMVDCVBVAJSDVEVLFVMVETUNVCVDVEUOUPUQURGEUS $.
   $}
 
+  ${
+    $d .0. b $.  $d .x. b $.  $d B b $.  $d I b $.  $d X b $.
+    prjspnfv01.f $e |- F = ( b e. B |->
+      if ( ( b ` 0 ) = .0. , b , ( ( I ` ( b ` 0 ) ) .x. b ) ) ) $.
+    prjspnfv01.b $e |- B = ( ( Base ` W ) \ { ( 0g ` W ) } ) $.
+    prjspnfv01.w $e |- W = ( K freeLMod ( 0 ... N ) ) $.
+    prjspnfv01.t $e |- .x. = ( .s ` W ) $.
+    prjspnfv01.0 $e |- .0. = ( 0g ` K ) $.
+    prjspnfv01.1 $e |- .1. = ( 1r ` K ) $.
+    prjspnfv01.i $e |- I = ( invr ` K ) $.
+    prjspnfv01.k $e |- ( ph -> K e. DivRing ) $.
+    prjspnfv01.n $e |- ( ph -> N e. NN0 ) $.
+    prjspnfv01.x $e |- ( ph -> X e. B ) $.
+    $( Any vector is equivalent to some vector whose value on the 0th axis is
+       either ` .0. ` or ` .1. ` : value part.  (Contributed by SN,
```

Comment:
Any vector is equivalent to a vector whose zeroth coordinate is .0. or .1. (proof of the value of the zeroth coordinate).